client/modules/IDE/components/Editor.jsx

```diff
@@ -197,7 +197,7 @@ class Editor extends React.Component {
             consoleEvent.data[0].indexOf(')') > -1) {
             const n = consoleEvent.data[0].replace(')', '').split(' ');
             const lineNumber = parseInt(n[n.length - 1], 10) - 1;
-            if (!Number.isNaN(lineNumber)) {
+            if (!Number.isNaN(lineNumber) && `${consoleEvent.source}.js` === this.props.file.name) {
```

Comment:
what if the error is in javascript that is in a script tag in the html file?

Reply:
I'll investigate what I can do in that case, but from what I found so far, there may be other fixes required to correctly report errors from javascript in a script tag anyway. When a runtime error is found in a <script> tag in the html, the filename is not reported and the line number is not reported correctly. I made a sketch to demonstrate: https://editor.p5js.org/jon_gualdarrama/sketches/NXb-s08x1

Reply:
Commit 39efd19 includes part of what I believe has to be done in order to highlight error lines in a <script> tag. It isn't complete yet, as the line number reported in the error will not match the line number in the html file on which the error actually occurred, so I set the line class on the correct line of the html file. After some googling, I've convinced myself that it has something to do with source maps - https://trackjs.com/blog/debugging-with-sourcemaps/. Will continue to investigate.

Reply:
i don't think it has to do with source maps—the sketch gets compiled into one long html file, which messes with the line numbers, and the necessary offsets are calculated by a function called `getAllScriptOffsets`, which is in `client/utils/consoleUtils.js`. looking at this function now, i see that it's only capturing the offsets for js files, and not inline js.